src/helper.py

```diff
@@ -152,6 +152,18 @@ def print_forecast(ocean, forecast):
             print("Wave Period: ", day[2])
         if int(ocean["show_uv"]) == 1:
             print("UV Index: ", day[4])
+        if int(ocean["show_air_temp"]) == 1:
+            print("Air Temp Max: ", day[5])
+        if int(ocean["show_air_temp"]) == 1:
+            print("Air Temp Min: ", day[6])
+        if int(ocean["show_air_temp"]) == 1:
+            print("Rain Sum: ", day[7])
+        if int(ocean["show_air_temp"]) == 1:
```

Comment:
Why do rain sum and precipitation probability show if `show_air_temp == 1`? We should create new variables, like `"show_rain_sum": 0` and let the user specify if they want to see it in a cli argument, like show_rain_sum/srs

See `set_output_values(args, arguments)` in `helper.py` for an example

Reply:
Also, if you make this change make sure to include it in the README's arguments section!

Reply:
I changed those while I was testing so they would show by default when I was adding them to the forecast... forgor to change them back.